src/tools/nnfusion/templates/dxcompute/Direct3DXBoxNN/DeviceResources.cpp

```diff
@@ -0,0 +1,444 @@
+//
+// DeviceResources.cpp - A wrapper for the Direct3D 12.X device and swapchain
+//
+
```

Comment:
Our code format change the order of this two headers, which causes a compiling error for xbox env. Can we just eliminate the code check under this template folder?

Reply:
sure